(just saving it here)

Fails to build on Android with:
```
[2023-05-15T17:05:22.981Z] sha1.c:97:22: warning: incompatible pointer types assigning to 'const unsigned int *' from 'ulong32 *' (aka 'unsigned long *') [-Wincompatible-pointer-types]
[2023-05-15T17:05:22.981Z] /nix/store/a4ji51giz7rdnlnpkya5rrq49alvvqlw-ndk-bundle-22.1.7171670/libexec/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/11.0.5/include/arm_neon.h:8540:47: note: expanded from macro 'vld1q_u32'
[2023-05-15T17:05:22.981Z] sha1.c:120:10: warning: implicit declaration of function 'vsha1h_u32' is invalid in C99 [-Wimplicit-function-declaration]
[2023-05-15T17:05:22.981Z] sha1.c:121:12: warning: implicit declaration of function 'vsha1cq_u32' is invalid in C99 [-Wimplicit-function-declaration]
[2023-05-15T17:05:22.981Z] sha1.c:121:10: error: assigning to 'uint32x4_t' (vector of 4 'uint32_t' values) from incompatible type 'int'
[2023-05-15T17:05:22.981Z] sha1.c:123:12: warning: implicit declaration of function 'vsha1su0q_u32' is invalid in C99 [-Wimplicit-function-declaration]
[2023-05-15T17:05:22.981Z] sha1.c:123:10: error: assigning to 'uint32x4_t' (vector of 4 'uint32_t' values) from incompatible type 'int'
...
[2023-05-15T17:05:22.981Z] fatal error: too many errors emitted, stopping now [-ferror-limit=]
```

I'll address this tomorrow, will probably have to narrow down the ARM detection bits

--- a/go.mod
+++ b/go.mod
@@ -10,7 +10,7 @@ replace github.com/nfnt/resize => github.com/status-im/resize v0.0.0-20201215164
 
 replace github.com/forPelevin/gomoji => github.com/status-im/gomoji v1.1.3-0.20220213022530-e5ac4a8732d4
 
-replace github.com/mutecomm/go-sqlcipher v0.0.0-20190227152316-55dbde17881f => github.com/status-im/go-sqlcipher v0.1.0-status.2
+replace github.com/mutecomm/go-sqlcipher v0.0.0-20190227152316-55dbde17881f => github.com/status-im/go-sqlcipher v0.1.0-status.3
 
 require (
 	github.com/anacrolix/torrent v1.41.0

